File: Java/SelectionSort.java
import java.util.*;
//selection sort with user input
public class SelectionSort {
    public static void main(String[] args) {
        Scanner in =new Scanner (System.in);
        int n=in.nextInt();
        int arr[]=new int[n];
        for (int i=0;i<n;i++){
            arr[i]=in.nextInt();
        }
        //selection sort
        for (int i=0;i<n-1;i++){
            int minInd=i;
            for (int j=i;j<n;j++){
                if (arr[j]<arr[minInd]){
                    minInd=j;
                }
            }
            int temp=arr[i];
            arr[i]=arr[minInd];
            arr[minInd]=temp;
        }
        //print the sorted array
        for (int e:arr){
            System.out.print(e+" ");
        }
    }


}

File: Python/selectionsort.py
def selection_sort(arr):
    n = len(arr)
    for i in range(n):
        min_idx = i
        for j in range(i+1, n):
            if arr[j] < arr[min_idx]:
                min_idx = j
        arr[i], arr[min_idx] = arr[min_idx], arr[i]
    return arr

# Get user input
user_input = input("Enter a list of numbers separated by spaces: ")
numbers = list(map(int, user_input.split()))

# Call the selection_sort function
sorted_numbers = selection_sort(numbers)

# Print the sorted list
print("Sorted list:", sorted_numbers)